mousestyles/data/__init__.py

```diff
@@ -62,3 +62,80 @@ def load_all_features():
     other_features = [x.iloc[:, -1] for x in data_frames]
     other_features.insert(0, all_features_df)
     return pd.concat(other_features, axis=1)
+
+
+def load_intervals(feature):
+    """
+    Parameters
+    ----------
+    feature: str
+             where feature is one of ["AS", "F", "IS", "M_AS", "M_IS", "W"]
+    Returns
+    -------
```

Comment:
This isn't quite the docstring format. See: https://github.com/berkeley-stat222/mousestyles/blob/master/DOCFLOW.md

I think it should look like

intervals: pandas.DataFrame
     <blah blah description here>

The same goes for the other function.